egs2/librispeech_100/asr1_biasing/conf/train_rnnt.yaml

```diff
@@ -0,0 +1,90 @@
+# Trained with A40 (48 GB) x 4 GPUs.
+encoder: conformer
+encoder_conf:
+    output_size: 256
+    attention_heads: 4
+    linear_units: 1024
+    num_blocks: 15
+    dropout_rate: 0.1
+    positional_dropout_rate: 0.1
+    attention_dropout_rate: 0.1
+    input_layer: conv2d
+    normalize_before: true
+    macaron_style: true
+    rel_pos_type: latest
+    pos_enc_layer_type: rel_pos
+    selfattention_layer_type: rel_selfattn
+    activation_type: swish
+    use_cnn_module: true
+    cnn_module_kernel: 31
+
+decoder: transducer
+decoder_conf:
+    rnn_type: lstm
+    num_layers: 1
+    hidden_size: 256
+    dropout: 0.1
+    dropout_embed: 0.2
+
+joint_net_conf:
+    joint_space_size: 320
+
+model_conf:
+    ctc_weight: 0.0
+    report_cer: False
+    report_wer: False
+    # Biasing
+    # Change this to the path of your biasing list
+    biasinglist: data/Blist/rareword_f15.txt
+    bmaxlen: 500
+    bdrop: 0.0
+    battndim: 256
+    biasing: true
+    biasingsche: 0
+    deepbiasing: true
+    # gcn for using GCN structure, 6 is the number of layers
+    biasingGNN: gcn6
+
+frontend_conf:
+    n_fft: 512
+    win_length: 400
+    hop_length: 160
+
+seed: 2022
+num_workers: 8
+batch_type: numel
+batch_bins: 6000000
+accum_grad: 8
+max_epoch: 70
+patience: none
+init: none
+best_model_criterion:
+-   - valid
+    - loss
+    - min
+keep_nbest_models: 10
+use_amp: false
+
+optim: adam
+optim_conf:
+    lr: 0.002
+    # weight_decay: 0.000001
```

Comment:
Since this hyperparameter is not used, will you delete it?

Reply:
Yes I will. Thanks!